test/unit/gateways/pin_test.rb

```diff
@@ -353,22 +346,12 @@ def test_add_creditcard_with_customer_token
     assert_false post.has_key?(:card)
   end
 
-  def test_add_3ds_v1
-    post = {}
-    @gateway.send(:add_3ds, post, @options.merge(three_d_secure: @three_d_secure_v1))
-    assert_equal '1.0.2', post[:three_d_secure][:version]
-    assert_equal '05', post[:three_d_secure][:eci]
-    assert_equal '1234', post[:three_d_secure][:cavv]
-    assert_equal '1234', post[:three_d_secure][:transaction_id]
-  end
-
-  def test_add_3ds_v2
+  def test_add_3ds
```

Comment:
Same as above, it would be good to retain these tests and add an additional test for the new style 3DS params.